pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -242,11 +190,11 @@ private void mapTissueToProperOncHistoryDetail(List<OncHistoryDetail> oncHistory
 
     void sortBySelfElseById(Collection<List<ParticipantData>> participantDatas) {
         participantDatas.forEach(pDataList -> pDataList.sort((o1, o2) -> {
-            Map<String, String> participantData = new Gson().fromJson(o1.getData().orElse(StringUtils.EMPTY),
-                    new TypeToken<Map<String, String>>() {
-                }.getType());
-            if (Objects.nonNull(participantData)
-                    && FamilyMemberConstants.MEMBER_TYPE_SELF.equals(participantData.get(FamilyMemberConstants.MEMBER_TYPE))) {
+            Map<String, String> participantData =
+                    new Gson().fromJson(o1.getData().orElse(StringUtils.EMPTY), new TypeToken<Map<String, String>>() {
```

Comment:
we introduced `ObjectMapperSingleton` class to replace `Gson` it would be good idea to eliminate `Gson` from the project, because during this time of refactoring we understood that `Gson` has some problems, for example it gets every number as a double, does not matter number is integer, long, etc, thus it made problems to us and we decided not to use it anymore, `ObjectMapperSingleton` uses `Jackson` which is better alternative of `Gson`